backend/import/do_import.py

```diff
@@ -63,8 +64,7 @@ def get_orgunit_data(ou):
 
 
 def get_employee_data(employee):
-    '''Get the data we need to display this employee'''
-
+    """Get the data we need to display this employee."""
```

Comment:
That's mostly in the function name, isn't it?

Reply:
You propose omitting this docstring, then?

Reply:
TBH I'm just complaining — I suppose something is better than nothing, but what's there doesn't really add much ;)